### PR TITLE
Images appearing in dev & prod

### PR DESCRIPTION
**Pull request by kim841**
# Pull Request

## Summary

After building the project, all images weren't appearing.
This solve the issue.

## Changes Made

- By moving the folder drawable from src/assets  to public/assets, all images appear (in dev & prod).
- Refactoring all their paths

## Related Issues

Solve #74 

## Checklist
Please ensure all the following steps are completed before submitting the pull request:

- [ ] Code passes all existing tests
- [ ] New features or changes are accompanied by appropriate tests
- [ ] Code follows the established coding style and conventions
- [ ] Documentation has been updated to reflect the changes (if applicable)
- [ ] All new and existing tests pass locally

### CODE REVIEW

File: android/.idea/deploymentTargetSelector.xml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <project version="4">
   <component name="deploymentTargetSelector">
     <selectionStates>
-      <SelectionState runConfigName="app">
+      <SelectionState runConfigName="Unnamed">
```
Comment (SarahHCode) [inline android/.idea/deploymentTargetSelector.xml:5]: Qu'est-ce runConfigName="Unnamed"?

File: tsconfig.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "lib": ["ESNext", "DOM"],
     "skipLibCheck": true,
     "noEmit": true,
+    "baseUrl": "./",
     "paths": {
           "@/*": [
```
Comment (SarahHCode) [inline tsconfig.json:17]: Veut-on enlever alias ici puisqu'on l'enlève pour les photos?

File: android/.idea/deploymentTargetSelector.xml (hunk 1 above)
Comment (kim841) [reply android/.idea/deploymentTargetSelector.xml:5]: dans android studio, 
image
il faudrait que je mette un nom plus beau haha

File: tsconfig.json (hunk 2 above)
Comment (kim841) [reply tsconfig.json:17]: bonne idée! je l'enlève

**Pull request merged**